### PR TITLE
added shh_getMessages, fixed #899

### LINKED ISSUE #899: calling web3.shh.watch().getMessages() is unimplemented
calling web3.shh.watch().getMessages() is unimplemented. the result is  "{code: -32601, message: "METHOD_NOT_FOUND: The method being requested is not available on this server"}"

### PR DESCRIPTION
**Pull request by debris**

### CODE REVIEW

File: test/webthreestubclient.h [hunk 1]
```diff
@@ -535,6 +535,16 @@ class WebThreeStubClient : public jsonrpc::Client
...
                 throw jsonrpc::JsonRpcException(jsonrpc::Errors::ERROR_CLIENT_INVALID_RESPONSE, result.toStyledString());
         }
+        Json::Value shh_getMessages(int param1) throw (jsonrpc::JsonRpcException)
+        {
+            Json::Value p;
+            p.append(param1);
+            Json::Value result = this->CallMethod("shh_getMessages",p);
+            if (result.isArray())
+                return result;
+            else
```
Comment (chfast) [inline test/webthreestubclient.h]: No `else` needed.

File: test/webthreestubclient.h [hunk 2]
```diff
@@ -535,6 +535,16 @@ class WebThreeStubClient : public jsonrpc::Client
             else
                 throw jsonrpc::JsonRpcException(jsonrpc::Errors::ERROR_CLIENT_INVALID_RESPONSE, result.toStyledString());
         }
+        Json::Value shh_getMessages(int param1) throw (jsonrpc::JsonRpcException)
+        {
+            Json::Value p;
+            p.append(param1);
+            Json::Value result = this->CallMethod("shh_getMessages",p);
```
Comment (chfast) [inline test/webthreestubclient.h]: Prefer `auto` over explicit type specification.

File: test/webthreestubclient.h [hunk 3]
```diff
@@ -535,6 +535,16 @@ class WebThreeStubClient : public jsonrpc::Client
             else
                 throw jsonrpc::JsonRpcException(jsonrpc::Errors::ERROR_CLIENT_INVALID_RESPONSE, result.toStyledString());
         }
+        Json::Value shh_getMessages(int param1) throw (jsonrpc::JsonRpcException)
```
Comment (chfast) [inline test/webthreestubclient.h]: throw list is deprecated in C++11.

File: test/webthreestubclient.h (hunk 1 above)
Comment (debris) [inline test/webthreestubclient.h]: webthreestubclient.h is autogenerated file

Comment (chfast) [inline test/webthreestubclient.h]: Great :)

**Pull request merged**